api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -75,6 +75,13 @@
 	protected static final String ENCOUNTER_OBS_XML = "org/openmrs/api/include/ObsServiceTest-EncounterOverwrite.xml";
 	
 	protected static final String COMPLEX_OBS_XML = "org/openmrs/api/include/ObsServiceTest-complex.xml";
+
+	@Test(expected = APIException.class)
+	@Verifies(value = "should throw APIException when obs is null", method = "saveObs(Obs,String)")
+	public void shouldReturnAPIExceptionWhenObsIsNull(){
+		ObsService os = Context.getObsService();
+		Obs o = os.saveObs(null,"Null Obs");
```

Comment:
How about you use a junit @Rule with ExpectedException? That way you check the message otherwise the exception could be thrown for something else since APIExceptions are thrown all over the code base.